other-attesting-env.txt

```diff
@@ -0,0 +1,27 @@
+  .--------------------------------.
+  |                                |
+  |            Verifier            |
+  |                                |
+  '--------------------------------'
+                          ^
+                          | 
+.-------------------------|-----------------------------------.
+|                         |                                   |
+|   .----------------.    |                                   |
+|   | Target         |    |                                   |
+|   | Environment    |    |                                   |
+|   |                |    | Evidence                          |
+|   '----------------'    |                                   |
+|                   |     |                                   |
+|                   |     |                                   |
+|          Collect  |     |                                   | 
+|           Claims  |     |                                   |
+|                   |     |                                   |
+|                   v     |                                   |
+|                 .-------------.             .-------------. |
+|                 | Attesting   |             | Other       | |
+|                 | Environment |<------------| Attesting   | |
+|                 |             |   Evidence  | Environment | |
```

Comment:
Is this Evidence different from the Evidence in the vertical arrow?
Assuming so, then `Evidence'` or `Evidence 2` would be more clear, but either way the text discussing the diagram should explain what happens with this Evidence, since right now I don't understand this diagram.